src/Sulu/Bundle/Sales/CoreBundle/Transition/TransitionManager.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Sulu\Bundle\Sales\CoreBundle\Transition;
+
+use Sulu\Bundle\Sales\CoreBundle\Entity\Transition;
+use Sulu\Bundle\Sales\CoreBundle\Entity\TransitionRepository;
+
+class TransitionManager
+{
+    /**
+     * @var TransitionRepository
+     */
+    protected $transitionRepository;
+
+    /**
+     * @param TransitionRepository $transitionRepository
+     */
+    public function __construct(TransitionRepository $transitionRepository)
+    {
+        $this->transitionRepository = $transitionRepository;
+    }
+
+    /**
+     * @param string $destinationClass
```

Comment:
i'd expect a class instead of a string, maybe $destinationAlias or $destinationClassName would be better